source/localizable/tutorial/deploying.md

```diff
@@ -1,4 +1,4 @@
-To deploy an Ember application simply transfer the output from `ember build` to a web server.
+To deploy an Ember application simply transfer the output from `ember build` to a web server (make sure to not build with `--enviroment=production`, as the data mocked with Mirage will only work with a development build).
```

Comment:
Could we tweak this a bit?

> simply transfer the output from `ember build --environment=development` to a web server (this ensures that Mirage continues to work instead of turning off as happens with `--environment=production`)

Thanks!